### ISSUE TITLE
[VSCRIPT] Add VScript Mapbase documentation

### ISSUE THREAD

**Issue by celisej567** [mapbase-source/source-sdk-2013#209]
Mapbase VScript has a lot of new stuff that other Source games don't have. Accordingly, you cannot find out about their existence if they are not written anywhere. I suggest creating a little VScript documentation with new features added only in Mapbase. This will help programmers understand what Mapbase VScript can do. Even just txt with some info about VScript can help.

**Comment by celisej567** [comment]
if you think script_help shows everything: no its not
![image](https://user-images.githubusercontent.com/75926135/197384776-a009c960-755d-4d7b-9017-f432c3d1410f.png)

**Comment by Jesse-Culver** [comment]
Agreed, I do not think anyone is in disagreement that vscript documentation could be better as a whole when it comes to Source. There is, I believe, a planned VScript sample repo so perhaps the documentation could be hosted there in the form of an .md? That way as Vscript coding changes occur someone can do a PR to update the documentation in addition to adding more samples.
